samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/OktaSdkHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Okta.Sdk;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace embedded_auth_with_sdk
+{
+    public static class OktaSdkHelper
+    {
+
+        public static async Task<string> ForgotPasswordGenerateToken(string email)
+        {
+            try
+            {
+                var client = new OktaClient();
+                var user = await client.Users.GetUserAsync(email);
+
+                var resetPassResponse = await user.ForgotPasswordGenerateOneTimeTokenAsync(false).ConfigureAwait(false);
+
+                return resetPassResponse?.ResetPasswordUrl?.Split('/').Last();
+            }
+            catch 
+            {
+                return null;
+            }
+        }

```

Comment:
Considering how few the lines of code are here, would it be easier for a user to follow if this code was put inline in the `AccountController.RecoverWithTokenAsync`?

Reply:
I'd like to have OktaSdk helper separated from controllers which work with Idx, also now we have a place to put all the Okta.Sdk work.

Reply:
I think I can 👀 see that argument 👍